gravatar-ui/src/main/java/com/gravatar/ui/SkeletonEffect.kt

```diff
@@ -0,0 +1,71 @@
+package com.gravatar.ui
+
+import androidx.compose.animation.animateColor
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.keyframes
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.foundation.background
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.rememberTextMeasurer
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun Modifier.skeletonEffect(): Modifier {
+    val infiniteTransition = rememberInfiniteTransition(label = "skeletonTransition")
+    val backgroundColor = infiniteTransition.animateColor(
+        // ⚠️ We should add the colors to the theme and use them here.
+        if (isSystemInDarkTheme()) Color(0xFF6B6B6B) else Color(0xFFE5E7E9),
+        if (isSystemInDarkTheme()) Color(0xFF979797) else Color(0xFFF2F2F2),
+        animationSpec = infiniteRepeatable(
+            animation = keyframes {
+                durationMillis = 800
```

Comment:
We should also move this to a `values.xml` file, that will allow third party to override it.

Reply:
Done [6370c0a](https://github.com/Automattic/Gravatar-SDK-Android/pull/123/commits/6370c0a53212cd332a1ad41bc6ecb98fa4389047)